### DISCUSSION TITLE
Getting HoughLine Data from within PerlMagick

### DISCUSSION THREAD

**Discussion by MikeTough** [ImageMagick/ImageMagick#5842] [Help] [answered]
Hi,

I'm using PerlMagick's HoughLine function, which is working well for my needs.  I note in ImageMagick it is possible to get an MVG text file with the numerical details of the lines identified.  However, in PerlMagick's documentation there is no mention of how to obtain this output within perl. 

Can anybody confirm if this is available in a perl environment, or if there is any work-around?

Mike

**Comment by urban-warrior** [comment, marked as answer]
Write the image in the MVG format, e.g. `WriteImage('image.mvg')` or `WriteImage('mvg:lines')`.

**Reply by MikeTough** [reply]
Ah, thanks for this. Writing out to *.mvg file and reading back into perl works a treat.
